node-wasm/src/wrapper/libp2p.rs

```diff
@@ -4,25 +4,33 @@ use libp2p::swarm::{
 use serde::{Deserialize, Serialize};
 use wasm_bindgen::prelude::*;
 
-#[wasm_bindgen]
+#[wasm_bindgen(inspectable)]
 #[derive(Debug, Serialize, Deserialize)]
 pub(crate) struct NetworkInfoSnapshot {
+    #[wasm_bindgen(js_name = "numPeers")]
     pub num_peers: usize,
-    connection_counters: ConnectionCountersSnapshot,
+    #[wasm_bindgen(js_name = "connectionCounters")]
+    pub connection_counters: ConnectionCountersSnapshot,
 }
 
-#[wasm_bindgen]
-#[derive(Debug, Serialize, Deserialize)]
+#[wasm_bindgen(inspectable)]
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
```

Comment:
How `inspectable` works? Can we remove `serde-wasm-bindgen` and use only that?

Reply:
> By default, structs exported from Rust become JavaScript classes with a single ptr property. All other properties are implemented as getters, which are not displayed when calling toJSON.

So I guess this primarily affects serialisation to json and debugging.

this is with `inspect`, you don't get the `toJSON` fn at all otherwise
![image](https://github.com/user-attachments/assets/1ccde8b8-4063-41fa-b9b5-0fc65664e276)